samples/java_layout/src/main/java/io/customer/android/sample/java_layout/data/model/CustomerIOSDKConfig.java

```diff
@@ -25,17 +25,16 @@ private static class Keys {
         static final String DEBUG_MODE = "cio_sdk_debug_mode";
     }
 
-
     public static CustomerIOSDKConfig getDefaultConfigurations() {
         return new CustomerIOSDKConfig(BuildConfig.SITE_ID,
                 BuildConfig.API_KEY,
-                BuildConfig.TRACKING_URL,
-                BuildConfig.BQ_SECONDS_DELAY,
-                BuildConfig.BQ_MIN_TASKS,
-                BuildConfig.ENABLE_IN_APP,
-                BuildConfig.TRACK_SCREENS,
-                BuildConfig.TRACK_DEVICE_ATTRIBUTES,
-                BuildConfig.DEBUG_MODE);
+                null,
+                null,
+                null,
+                null,
+                null,
+                null,
```

Comment:
I would prefer `null` over SDK constants because having non-null value means we are setting all these configurations every time in our sample app. Having `null` values helps us validate default behavior of SDK where customers do no set these values explicitly.

Reply:
🤔 I am wondering if instead of trying to construct a new SDK config object like we are doing here, we can instead reference the SDK config object currently set in the CustomerIO SDK? 

[I made a similar change](https://github.com/customerio/customerio-ios/pull/298) on iOS where you can now call `CustomerIO.shared.config` to get an SDK config object with the currently set values on it for you to then reference or use to modify the SDK. 

Would it be worth it to make a similar change on the Android SDK? Be able to call `CustomerIO.instance().config`?

Reply:
I don't like the idea yet. Mainly because customers will generally only be looking to update one or a few values, where current builder pattern already does a good job. Also, I'm not sure if updating any value in config at runtime will be utilized properly by the SDK or not.

So this definitely needs some testing and outside the scope of this ticket, we can discuss and make the changes in another issue if we want. But for this ticket, I think there isn't any action item for this.

Reply:
>`CustomerIO.instance().config`

Sounds like a good thing to have but not very useful in this regard. because we are going to persist in the configurations anyways, to make sure they stay the same when app relaunches. 

>I would prefer null over SDK constants because having non-null value means we are setting all these configurations every time in our sample app

Internally that's what the SDK is doing as well, setting the same defaults because the source will be same. But if you want to stick with `null`, thats not a blocker.